scripts/check_js_intergation.sh

```diff
@@ -13,7 +13,12 @@ ROOT=`pwd`
 npm -v
 echo "starting containers..."
 cd $ROOT/test/bdd/fixtures/agent-wasm
-(source .env && docker-compose down && docker-compose up --force-recreate -d)
+
+(source .env && docker-compose down --remove-orphans && docker-compose -f docker-compose.yml up --force-recreate -d)
+
+bash generate_config.sh
```

Comment:
Why we need it only for bddtests and not for user-agent-start?

Reply:
fixed.

Reply:
not sure what did you fix here

Reply:
I added this to scripts/user_agent_start.sh, so now user-agent-start also splits the docker-compose and generates the configs.